sdk/textanalytics/Azure.AI.TextAnalytics/tests/samples/Sample2.1_AnalyzeSentimentWithOpinionMining.cs

```diff
@@ -22,18 +22,34 @@ public void AnalyzeSentimentWithOpinionMining()
             var client = new TextAnalyticsClient(new Uri(endpoint), new AzureKeyCredential(apiKey));
 
             #region Snippet:TAAnalyzeSentimentWithOpinionMining
+            string reviewA = @"The food and service were unacceptable, but the concierge were nice.
+                             After talking to them about the quality of the food and the process
+                             to get room service they refunded the money we spent at the restaurant
+                             and gave us a voucher for near by restaurants.";
```

Comment:
I can be wrong, but I think it should be `nearby`. Not that it matters much... it's an informal review, anyway. Same comment applies to other places where this document has been used.

Reply:
Thank you!